File: 1603-RunningSumOf1dArray/1603-RunningSumOf1dArray.py
# Last updated: 4/29/2026, 10:23:31 AM
class Solution:
    def runningSum(self, nums: List[int]) -> List[int]:
        sumList = []
        sum = 0
        for num in nums:
            sum += num
            sumList.append(sum)
        return sumList
    

File: 1603-RunningSumOf1dArray/1603-RunningSumOf1dArray.js
// Last updated: 4/29/2026, 10:23:36 AM
/**
 * @param {number[]} nums
 * @return {number[]}
 */
var runningSum = function(nums) {
    const sums = [];
    let sum = 0;
    for(const num of nums) {
        sum += num;
        sums.push(sum);1
    }
    return sums;
};